stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1640,7 +1640,7 @@ ParseResult ReduceOp::parse(OpAsmParser& parser, OperationState& result) {
         parser.parseKeyword("reducer"))
       return failure();
     OpBuilder builder(parser.getBuilder().getContext());
-    result.addAttribute("dimensions", builder.getI64TensorAttr(dimensions));
+    result.addAttribute("dimensions", builder.getDenseI64ArrayAttr(dimensions));
```

Comment:
Should we make this change now or when we flip the default to DenseArray? Same below.